README.md

```diff
@@ -0,0 +1,10 @@
+# DWYL Elm MVP application
+
+MVP application build with Elm
+
+# Run it
+
+- Clone this repository: `git clone git@github.com:dwyl/app-mvp-elm.git && cd app-mvp-elm`
+- Install elm-live, see https://github.com/wking-io/elm-live
+- Run `elm-live src/Main.elm --pushstate -- --outpute elm.js --optimize` see https://github.com/dwyl/learn-elm/issues/156
+- Visit localhost:8000 to see the application
```

Comment:
consider adding more docs on what the person should _expect_ to see when they boot the App. 💭